reasoner/lube-logical/src/test/scala/com/antgroup/openspg/reasoner/lube/logical/OptimizerTests.scala

```diff
@@ -63,101 +63,6 @@ class OptimizerTests extends AnyFunSpec {
     }
   }
 
-  it("testEdgeToProperty") {
```

Comment:
why delete this unit test? Is thers a substitute unit test?

Reply:
Since there are still some bugs in the implementation of EdgeToProperty, i will write unit test code for the EdgeToProperty optimization rule after subsequent improvements.